engines/python/setup/djl_python/rolling_batch/scheduler_rolling_batch.py

```diff
@@ -143,7 +143,13 @@ def _prefill_and_decode(self, new_requests):
         for request_id, generated_token, request in zip(
                 request_ids, generated_tokens, self.pending_requests):
             is_last_token = (request_id in exit_req_ids)
-            request.set_next_token(generated_token, last_token=is_last_token)
+            if self.output_formatter is not None:
```

Comment:
request.set_next_token(generated_token, self.output_formatter, last_token=is_last_token)

Reply:
Do not format output for scheduler_rolling_batch?

Reply:
Changed